scripts/views/repCardView.js

```diff
@@ -30,6 +30,7 @@
   repCardHandler.renderRepresentativeCards = function(representativePromise, $parent, state) {
     $parent.empty(); // If they search for a new zipcode clear the old info
     representativePromise.then(function(representatives) {
+      console.log(representatives);
       var compiledTemplate = Handlebars.getTemplate('representativeCard');
```

Comment:
can you remove this log?

Reply:
Ooops!  Fixed.

Reply:
it looks like it's still there

Reply:
Gah!  Forgot to stage during my amend commit.